code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -3380,30 +3380,30 @@
 	icon_state = "cutlet"
 	bitesize = 2
 	center_of_mass = list("x"=17, "y"=20)
-	preloaded_reagents = list("protein" = 2)
+	preloaded_reagents = list("protein" = 3)
 	taste_tag = list(MEAT_FOOD,SPICY_FOOD)
 
 /obj/item/reagent_containers/food/snacks/rawmeatball
 	name = "raw meatball"
 	desc = "A raw meatball."
 	icon = 'icons/obj/food_ingredients.dmi'
 	icon_state = "rawmeatball"
-	bitesize = 2
+	bitesize = 3
 	center_of_mass = list("x"=16, "y"=15)
-	preloaded_reagents = list("protein" = 2)
+	preloaded_reagents = list("protein" = 3, "flour" = 5)
```

Comment:
that's an exact chem value required to make meatballs, are you sure this won't make them pop out of nowhere? Did you test it?